src/libaktualizr/crypto/keymanager.h

```diff
@@ -42,12 +42,10 @@ class KeyManager {
  private:
   const std::shared_ptr<INvStorage> &backend_;
   KeyManagerConfig config_;
-#ifdef BUILD_P11
-  P11EngineGuard p11_;
-#endif
-  std::unique_ptr<TemporaryFile> tmp_pkey_file;
-  std::unique_ptr<TemporaryFile> tmp_cert_file;
-  std::unique_ptr<TemporaryFile> tmp_ca_file;
+  std::unique_ptr<P11EngineGuard> p11_{nullptr};
+  std::unique_ptr<TemporaryFile> tmp_pkey_file{nullptr};
+  std::unique_ptr<TemporaryFile> tmp_cert_file{nullptr};
+  std::unique_ptr<TemporaryFile> tmp_ca_file{nullptr};
```

Comment:
Is the explicit initialization to `{nullptr}` needed?  Or is this a stylistic thing?

Reply:
They should not be needed but I added them to fix an "uninitialized attribute" warning and don't remember in which configuration it showed up. I'll try removing these and see what happens